jep/1/README.adoc

```diff
@@ -256,21 +256,27 @@ let's take a high-level look at how JEP might go.
 
 . **<<submission, Submission>>** - Andrea writes up the proposal using the JEP document template as a guide.
   She includes supporting documentation
-  and a prototype implementation sufficient to convey the viability of the design.
+  and a minimal prototype implementation sufficient to convey the viability of the design.
   She submits the JEP to the JEP editors for
   <<approval, approval as a Draft JEP>>.
   An editor checks the submission and determines it is ready to considered as a JEP.
   They "approve" the submission, assigning the JEP a number, and the submission becomes a "Draft" JEP.
 
-. **<<draft, Draft Status>>** - While the JEP is a "<<draft, Draft>>", Andrea may continue to gather
+. **<<draft, Draft Status>>** - While the JEP is a "<<draft, Draft>>", Andrea continues to gather
   feedback, change the proposal, and record the reasoning and differing views.
-  When she believes the design is complete and represents the consensus of the community,
-  she submits the JEP for review.
-
-. **<<review, Review>>** - The <<Reviewer>> for this JEP,
-  reviews the JEP and decides to accept it, making it an "<<accepted, Accepted>>" JEP.
-  Other possible resolutions are "<<rejected, Rejected>>",
-  "<<deferred, Deferred>>", "<<withdrawn, Withdrawn>>".
```

Comment:
Hmm, why do you remove them? The ML proposes to return to Draft instead of rejection, but Withdrawn/Deferred are still possible outcomes

Reply:
@oleg-nenashev  I've added an "Other outcomes" section to the overview to preserve the mention of these statuses independent of Review.